IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoadDomain/PropertySets/Pset_BoundedCourseCommon/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_BoundedCourseCommon" UniqueId="75246832-754b-4570-a4e2-24baa7a30a33" ApplicableType="IfcCourse" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_BoundedCourseCommon" UniqueId="75246832-754b-4570-a4e2-24baa7a30a33" ApplicableType="IfcCourse,IfcCourseType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
I am a bit skeptical about `IfcObjectType` applicabilities. These were supposed to be handled on occurrence level. If this is supposed to change, sure but we need a common agreement on this.

Reply:
@SergejMuhic @larswik needs resolving